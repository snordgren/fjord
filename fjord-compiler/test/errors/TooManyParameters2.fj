module test.errors.TooManyParameters2

myFun : Int -> Int -> Int
myFun a b c =
  a
