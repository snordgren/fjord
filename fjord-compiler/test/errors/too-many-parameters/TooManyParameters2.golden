test/errors/too-many-parameters/TooManyParameters2.fj:4:11:
  |
4 | myFun a b c =
  |           ^^^^
too many parameters, expected 2
